### DISCUSSION TITLE
manually and permanently compile certain classes?

### DISCUSSION THREAD

**Discussion by fredbouin** [tailwindlabs/tailwindcss#8164] [Help] [answered]
I want to give users the ability to pick a css grid column span through my cms control panel. My approach is to have a fieldset called ```col-span``` with a range picker from 1 to 12 (for a 12 column grid). Then in the html I simply do something like this:
```
<div class="col-span-{{ col_span }}">stuff</div>
```

This would work fine except ```col-span-{{ col_span }}``` won't compile into anything so in the end even if the resulting css looks like ```col-span-6``` tailwind doesn't understand it because it hasn't been compiled.

Could I possibly manually set all col-span options 1 through 12 as permanently compiled?

**Comment by onmax** [comment, marked as answer]
I think what you are looking for is the safelist in tailwind.config.js. See docs: https://tailwindcss.com/docs/content-configuration#safelisting-classes

**Reply by fredbouin** [reply]
perfect, this is what i needed
